services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java

```diff
@@ -30,6 +30,7 @@
 import org.keycloak.models.RealmModel;
```

Comment:
@douglaspalmer what happened with the endpoint to remove applications? It seems that it was removed by accident.